civiform/civiform
# Improvements to the focus outline

### Description

Added a white offset between interactive elements and the focus outline.  This is especially helpful for color contrast around blue buttons since our focus outline is also blue.

Also, on checkboxes and radio buttons, moved the focus outline to the label rather than the button itself.

### Instructions for manual testing

- Notice the offset between interactive elements and the focus outline.  For example:

![image](https://github.com/civiform/civiform/assets/134556723/68ea9797-fe08-4b66-9592-f34d46f8016d)

- Check radio button and checkbox questions in an application.  Make sure the focus outline is on the label rather than the checkbox or button.

![image](https://github.com/civiform/civiform/assets/134556723/033d6ebd-bdc3-41fa-bd1a-594fa852eb66)

- Also check the radio buttons when editing a program:

![image](https://github.com/civiform/civiform/assets/134556723/901abb4e-170d-4593-85b0-dd9bd5ebfe3c)

### Issue(s) this completes

Fixes #5947
